apps/mark-scan/frontend/src/pages/pat_device_identification/pat_introduction_step.tsx

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useCallback } from 'react';
+import { H1, Icons, P, Text } from '@votingworks/ui';
+import { validKeypressValues } from './constants';
+import { PortraitStepInnerContainer } from './portrait_step_inner_container';
+
+export function PatIntroductionStep({
+  onStepCompleted,
+}: {
+  onStepCompleted: () => void;
+}): JSX.Element {
+  const handleInput = useCallback(
+    (event: KeyboardEvent) => {
+      if (validKeypressValues.includes(event.key)) {
+        event.preventDefault();
+        onStepCompleted();
+      }
+    },
+    [onStepCompleted]
+  );
+
+  useEffect(() => {
+    document.addEventListener('keydown', handleInput);
+
+    return () => {
+      document.removeEventListener('keydown', handleInput);
+    };
+  });
+
+  return (
+    <PortraitStepInnerContainer>
+      <Icons.Info />
+      <H1>Test Your Device</H1>
+      <P>This voting machine has two functions for device inputs:</P>
+      <P>
+        <ol>
+          <li>
+            Navigate Focus eg. move screen focus between candidates in a contest
+          </li>
+          <li>
+            Activate Focus eg. activate the focused candidate&apos;s row to mark
+            a vote for them
+          </li>
+        </ol>
+      </P>
+      <P>
+        The device test will ask you to trigger both device inputs in turn to
+        identify their functions. After both inputs are identified you will be
+        returned to the previous screen.
+      </P>
```

Comment:
Some copy suggestions:
- Use "Navigate" and "Activate" without the word focus to be consistent with the other screens.
- I know we've bike-shedded the specific terms a lot, but I liked "Select" for being a simpler word than "Activate". Maybe "Move" and "Select" are the two simplest terms I can think of. But I don't feel strongly.
- People often skim instructions so maybe trim them down

So maybe something like:

**Test Your Device**
Your device's two inputs can be used to **Move** focus between items on the screen and **Select** an item.

**Trigger any input to begin the test.**

Reply:
Agree with @jonahkagan on copy simplification suggestions. Particularly use of the words "move" and "select".